jobs/firstinqueue-priming.groovy

```diff
@@ -99,42 +95,17 @@ def _setupWebapp() {
 
 def prime() {
    _setupWebapp();
-   // We also need to sync mobile, so we can run the mobile integration test
-   // (if we are assigned to do so).
-   // Note that Mobile latest code is at develop branch
-   // TODO(benkraft): Only run this if we get it from the splits?
-   kaGit.safeSyncToOrigin("git@github.com:Khan/mobile", "develop");
```

Comment:
**note:** These mobile integration tests are not supported by our Cypress infra/tests. If we still need to keep that around, we probably need to switch those tests to a different testing approach (e.g. integration tests in Go?).

Reply:
Cc @jaredly and @jeresig , who I think set this up originally.  What do we want to do with legacy_e2e/activity/exercises/end_to_end/android_perseus_smoketest.py?  There's no analogue in cypress-land.

Though it looks like the test is currently disabled.  So maybe it's ok just to get rid of it?

Reply:
Yeah, I'm fine just getting rid of it!